Sources/PrimerSDK/Classes/Extensions & Utilities/NSObject+ClassName.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  NSObject+ClassName.swift
+//  PrimerSDK
+//
+//  Created by Dario Carlomagno on 07/06/22.
+//
+#if canImport(UIKit)
+
+import Foundation
+
+extension NSObject {
```

Comment:
That's a bit tricky. I believe this functionality gets exposed even if the access level is internal. It does not get exposed through the extension, but it will get exposed when someone subclasses `NSObject`.

Can you make the following test, in the Example app create a subclass of `NSObject` and check whether those vars are exposed?

If they are, try subclassing `NSObject`, extend your subclass with those vars, then use your subclass instead of `NSObject`.

Reply:
Thanks for the inputs 💪 - checked. I believe we are good. (Screenshot from a swift file within the example app)

<img width="1105" alt="Screen Shot 2022-06-14 at 17 21 58" src="https://user-images.githubusercontent.com/3584008/173615222-0fc40af5-8f6a-4e55-b5af-6d153c98de6b.png">